### ISSUE TITLE
Start crawl job end point

### ISSUE THREAD

**Issue by oliversauter** [WorldBrain/cortex#10]
Following the [architecture diagram][1], we need:

A single endpoint which looks like:

```
POST /api/crawler/
```

Which accepts a body of:

```
{'domains': [domain1, domain2, domain3, domain4]}
```

Which will do the following:

  - validate the domains are really valid domains ([docs][2])
  - make a call to the crawler (doesn't exist yet, but we can mock it)

This will then send a single request to the crawler which starts a crawl job.

[1]: https://github.com/WorldBrain/project-management/issues/24#issue-167050314
[2]: https://docs.djangoproject.com/en/1.9/ref/validators/#urlvalidator

**Comment by oliversauter** [comment]
@hoshomoh is this already done with your setup of django?

**Comment by oliversauter** [comment]
done? @hoshomoh

**Comment by hoshomoh** [comment]
I dont think this is done. Maybe @vpolisky or @tobisanya would be able to confirm this.
